apps/coscrad-frontend/src/store/slices/resources/shared/connected-resources/connected-resources-panel.spec.tsx

```diff
@@ -60,7 +60,9 @@ const connectedTerm = dummyTerms[1];
 const connectedBook = buildDummyBooks()[0];
 
 const connectedSpatialFeature = buildDummySpatialFeatures()[0];
-
+/**
+ * Be sure to update this test to work on the new approach.
```

Comment:
Do this.

Reply:
It doesn't look like we need to change the test to support the refactor (which is good).